app/overrides/admin_decorator.rb

```diff
@@ -0,0 +1,4 @@
+Deface::Override.new(:virtual_path => "spree/admin/shared/sub_menu/_configuration",

```

Comment:
Line is too long. [83/80]<br>Carriage return character detected.<br>Use the new Ruby 1.9 hash syntax.